src/engine.ts

```diff
@@ -3,6 +3,7 @@ import { clean as removeDiacritic } from 'diacritic';
 const punct = [
     '\\p{P}', // All and every possible punctuation characters
     '\\p{Z}', // All (almost) and every possible whitespace characters
+    '\\n', // New line
```

Comment:
Please, move this item on top of the list or at the bottom, but not in the middle of emoji-related expressions.

Reply:
Fixed